---
title: "Brian C. O'Meara CV"
author: "bomeara@utk.edu"
date: today
date-format: medium
format: pdf
---

```{r, echo=FALSE, cache=TRUE}
library(scholar)
library(rorcid)
library(rcrossref)
library(cranlogs)


### From bibr: https://github.com/edonnachie/bibr/blob/master/R/doi_parse.R

#' https://github.com/regexhq/doi-regex
#' http://www.doi.org/doi_handbook/2_Numbering.html#2.2
doiRegex <- '(10[.][0-9]{4,}(?:[.][0-9]+)*/(?:(?![%"#? ])\\S)+)'

#' Test whether string contains a doi
#'
#' @param string String to test
#' @return Logical indicator TRUE (matches doi specification) or FALSE
#' @export
is_doi <- function(string) {
  grepl(doiRegex, string, perl=TRUE)
}


#' Extract all DOI from a character object
#'
#'
#' @param string String from which to extract all doi entries
#' @param as_vector A single logical value. If TRUE (default), calls `unlist` to return to a single character vector. If FALSE, returns a list of character vectors, one for each element of `string`
#' @export
#' @importFrom stringi stri_extract_all
extract_doi <- function(string, as_vector = TRUE) {
  out <- stringi::stri_extract_all_regex(string, pattern = doiRegex)
  if (as_vector)
    out <- unlist(out)
   out <- doi_strip_url(unique(gsub(',','',out))) # This line by Brian O'Meara
  return(out)
}


#' Remove the URL part (i.e. "http://dx.doi.org/") from a doi
#'
#' @param doi DOI, with or without URL part
#' @return string with URL part removed
#' @export
doi_strip_url <- function(doi){
  sub("http://dx.doi.org/", "", doi)
}

try_unname_unlist <- function(x) {
	result <- ""
	try({
		result <- unname(unlist(x))
	}, silent=TRUE)
	return(result)
}

unique_no_NA <- function(x) {
	unique(x[!is.na(x)])
}

scholar.id="vpjEkQwAAAAJ"
orcid.id="0000-0002-0337-5997"
g.profile <- scholar::get_profile(scholar.id)
scholar_pubs <- scholar::get_publications(scholar.id)
orcid_pubs <- rorcid::orcid_works(orcid.id, format="application/json")[[1]][[1]]

orcid_pubs$dois <- NA
for (i in sequence(nrow(orcid_pubs))) {
	all_dois <- unique_no_NA(c(extract_doi(orcid_pubs$url.value[i]), extract_doi(try_unname_unlist(orcid_pubs$`external-ids.external-id`[i][[1]][,'external-id-value']))))
	if(length(all_dois) > 0) {
		orcid_pubs$dois[i] <- all_dois[1]
	}
}

orcid_pubs <- orcid_pubs[!is.na(orcid_pubs$dois),]
orcid_pubs <- orcid_pubs[!duplicated(orcid_pubs$dois),]

preprint_indices <- c(which(grepl("xiv", orcid_pubs$`journal-title.value`, ignore.case=TRUE)), which(is.na(orcid_pubs$`journal-title.value`)))


orcid_pubs_preprints <- orcid_pubs[preprint_indices,]
orcid_pubs_not_preprints <- orcid_pubs[-preprint_indices,]


publications_formatted <- rcrossref::cr_cn(orcid_pubs_not_preprints$dois, format='text', style='apa')

preprints_formatted <- rcrossref::cr_cn(orcid_pubs_preprints$dois, format='text', style='apa')

ouwie <- sum(cran_downloads(package = "OUwie", from = "2001-01-01", to = Sys.Date())$count, na.rm = TRUE)
hisse <- sum(cran_downloads(package = "hisse", from = "2001-01-01", to = Sys.Date())$count, na.rm = TRUE)
corHMM <- sum(cran_downloads(package = "corHMM", from = "2001-01-01", to = Sys.Date())$count, na.rm = TRUE)

# lots not on CRAN, too, and this only tracks rstudio

funding_amounts <- c(193401,738297,148101, 520000, 340000 , 393074 , 98252 ,138590 , 132345)
total_funding <- sum(funding_amounts)

```

# Overview

* **Website**: [brianomeara.info](https://brianomeara.info)
* **Email**: bomeara@utk.edu
* **Github**: [github.com/bomeara](https://github.com/bomeara)
* **Research**: Evolutionary biology, with a focus on phylogenetics and macroevolution
* **Teaching**: Macroevolution, Core Evolution, Phylogenetic Methods
* **Publications**: `r nrow(orcid_pubs_not_preprints)` publications, `r g.profile$total_cites` citations, h-index of `r g.profile$h_index`
* **Funding**: $`r formatC(total_funding, format="d", big.mark=",")` in external funding from `r length(funding_amounts)` grants
* **Mentoring**: Mentored 17 postdocs, 8 grad students (including 2 current), 5 undergrad students, 4 faculty, and served on numerous grad student committees
* **Service**: President of the Society of Systematic Biologists, Associate Head of EEB, Associate Director of NIMBioS, Code of Conduct for three evolution societies, and much more

# Research

I address questions in evolutionary biology through development, implementation, and application of new phylogenetic methods. These include approaches for examining the process of continuous trait evolution (rate, optimal values, and other factors), species delimitation, phylogeography, diversification analyses, biogeography, and more. I collaborate extensively with empiricists, including some of my own students. Many of my approaches are made usable in popular software collaborators and I write; for example, my R packages have over `r formatC(1000*floor((1/1000)*(ouwie + hisse + corHMM)), format="d", big.mark=",")` downloads.

# Education

* University of California Davis: PhD (2008) in Population Biology. Advisors: Mike Sanderson, Phil Ward, Michael Turelli

* Harvard University: Bachelors (magna cum laude), with highest honors in
Biology (2001)

# Employment

* 2019-Present: **Professor**, Dept. of Ecology & Evolutionary Biology, University of Tennessee, Knoxville, TN

* 2016-2022: **Associate Head**, Dept. of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, TN

* 2016-2017: **Associate Director**, National Institute for Mathematical and Biological Synthesis (NIMBioS), University of Tennessee, Knoxville, TN

* 2015-2019: **Associate Professor**, Dept. of Ecology & Evolutionary Biology, University of Tennessee, Knoxville, TN

* 2009-2015: **Assistant Professor**, Dept. of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, TN

* 2007-2009: **Postdoc**, National Evolutionary Synthesis Center, Durham, NC

# Publications

According to Google Scholar, my work has been cited `r g.profile$total_cites` times, and my h-index is `r g.profile$h_index`. I use a stringent criterion for when I get authorship -- I have to actively make a significant contribution to the research and writing to merit authorship (for example, not all postdoc or grad student projects in my lab have me as a coauthor, even if I helped with feedback or ideas -- perhaps a bit annoying in the year people in my lab got three different *Science* papers without me as a coauthor, but still I think an honest way to do science). 


`r paste(unlist(publications_formatted), collapse = "\n\n")`


# Preprints

`r paste(unlist(preprints_formatted), collapse = "\n\n")`

# Teaching

Our standard load is to have three substantial courses and one small course (like a reading group) over a two year period (but teaching every semester). I have been teaching slightly over this (with my agreement) for the past few years.

## Current

* Macroevolution (EEB 464): An undergraduate or early grad student course on "macroevolution" which includes everything from spandrels to genomes. I created this course as an assistant professor; its enrollment has grown and now has 40 students. I currently teach it every year; its modality is asynchronous online. I have a mixture of recorded lectures, readings, videos from others, software demos, and more.
* Core Evolution (EEB 509): An introduction to evolution: required for all grad students in UTK EEB, and often taken by students in related departments. It fulfills part of the background knowledge part of their candidacy exam. Each session is typically a mixture of a brief introductory lecture to a topic, then a longer full class discussion (including think-pair-share) of one or more papers previously assigned on that topic. Typically 10-20 students.

## Past

* PhyloMeth: Partially supported by an NSF CAREER grant, a class that focuses on phylogenetics, with a focus on methods for using trees. The capstone was students developing a method (tuned to the starting level of the students). Materials are available on <phylometh.info>. 
* Speciation reading group: Grad students, postdocs, and faculty meet weekly to discuss papers on speciation (or related topics).
* Introduction to Biology (BIO 130 or 150): Large (150-200 student) enrollment classes. 
* Various workshops on comparative methods and R (e.g., at NIMBioS, NESCent, Friday Harbor, Mathematical Biosciences Institute (MBI), scientific conferences, etc.). Easily over a thousand students have taken these workshops.
* Group lab meetings (3-4 evolutionary biology labs meeting jointly), counted as a graduate seminar. 

# External Funding

While my lab infrastructure is fairly cheap (individual computers, computing clusters, whiteboards, and markers), paying humanely for people isn't. Fortunately, I have been able to secure $`r formatC(total_funding, format="d", big.mark=",")` funding for my institution (see below). As with my listing of publications, this does not include funding earned by students or postdocs in my lab (GRFPs, DDIGs, postdoc fellowships, research funds, etc.) -- those go on their CVs. Funding indicated is that going to my institution, not necessarily the total amount of the grant.

* 2019-2024: "Collaborative Research:  Novel framework for estimating continuously-varying diversification rates." NSF DEB, Systematics & Biodiversity Science. $193,401
* 2015-2021: "CAREER: Reducing barriers for comparative methods." NSF DEB, Phylogenetic Systematics. $738,297
* 2015-2019: "Collaborative Research: ABI Development: An open infrastructure to disseminate phylogenetic knowledge." NSF DBI, Advances in Bioinformatics. $148,101
* 2014-2019: "Population genetics-based codon models." NSF DEB, Genetic Mechanisms, Information Technology Researc, Cross-BIO Activities, Systematics & Biodiversity Sci, Algorithmic Foundations. $520,000
* 2013-2017: "Collaborative Research: Phylogeographic Inference Using Approximated Likelihoods." NSF DEB, Phylogenetic Systematics. $340,000.
* 2012-2017: "Historical naming traditions and cryptic speciation bias biodiversity estimates in transatlantic agaric fungi." NSF DEB, Biodiversity: Discovery & Analysis. $393,074
* 2012-2013: "rPlant". Subaward from an NSF $50,846,500 grant. $98,252 to UTK
* 2011-2012: "iPlant: Trait evolution group, year 2". Subaward from an NSF $50,846,500 grant. $138,590 to UTK
* 2010-2011: "iPlant: Trait evolution group, year 1". Subaward from an NSF $50,846,500 grant. $132,345 to UTK

# Mentoring

Here are some of the people I have mentored; I am not including graduate students on whose committees I have served (probably dozens by this point, across multiple departments at UTK and as an external member at other institutions).

## Postdocs

|Name                                                                             |Duration     |NIMBioS |
|:--------------------------------------------------------------------------------|:------------|:-------|:---------------------------------------------------------------|
|[Hugo Alamillo](https://people.northseattle.edu/users/halamillo)                 |2011-2012    |N                           |
|[Barb Banbury](https://www.linkedin.com/in/bbanbury)                             |2010-2012    |N                                 |
|[David Bapst](https://geogeo.tamu.edu/people/profiles/faculty/bapstdavidwilliam.html)  |2017-2019    |N                    |
|[Jeremy Beaulieu](https://www.jeremybeaulieu.org/)                                |2012-2016    |Both                              |
|[James Boyko](https://jamesboyko.com/)                                |2022-2023   |N                               |
|[Juanjuan (JJ) Crosskey (formerly Chai)](https://www.linkedin.com/in/jjcrosskey) |2011-2013    |Y                  |
|[Dominic Evangelista](https://www.roachbrain.com)                                |2018-2018    |Y                              |
|[Nathan Jackson](https://nathandjackson.com/)                                     |2013-2016    |N                            |
|[Tony Jhwueng](https://www.tonyjhwueng.info/)                                     |2009-2011    |Y                              |
|[Sandy Kawano](https://sandykawano.weebly.com/)                                   |2014-2016    |Y       |
|[Michelle Lawing](https://people.tamu.edu/~alawing/)                              |2012-2014    |Y                                         |
|[Ryan Martin](https://www.martinevolutionaryecologylab.com)                       |2012-2013    |Y                                    |
|[Nick Matzke](https://nickmatzke.weebly.com)                                      |2013-2015    |Y         |
|[Megan Rua](https://meganrua.wordpress.com)                                      |2015-2016    |Y                              |
|[Luna Sanchez Reyes](https://www.lunasare.com/)                                  |2017-2019    |N                |
|[Sergei Tarasov](https://www.tarasovlab.com/)                |2016-2018    |Y          |
|[Jodie Wiggins](https://jwiggi18.github.io/)                                     |2018-2019    |N                      |

## Grad students in my lab

These are students for whom I was their advisor or co-advisor. Note that I include just the time they were in my lab, not the time they were in the program. Our students must have one or more advisors when they enroll, but they may also add or replace advisors. All but one of the students below started in a different lab and then added me; so far, the reverse has not happened (but I encourage any of my students reading this CV to switch if it helps you with your goals).

|Name                                                                      |Stage       |Time in Lab  |Note                           |
|:-------------------------------------------------------------------------|:-----------|:------------|:------------------------------|
|[Sam Borstein](https://fish4thought.wordpress.com/)                       |PhD student |2014-2019    |                               |
|[Jenn Bosco](https://sites.google.com/site/jenniferboscoinfo/home)        |PhD student |2012-2017    |Co-advised with Susan Riechert |
|[Amanda Chandler](https://www.linkedin.com/in/amanda-chandler-90045b1b9/)                       |Masters student |2024-present    | 
|[Krista De Cooke](https://www.linkedin.com/in/krista-de-cooke-163a87116/) |Masters student |2020-2023 |Co-advised with Dan Simberloff |
|[Katie Massana](https://sites.google.com/site/kathrynamassana/home)       |PhD student |2012-2017    |        
|[Alivia Nytko](https://alivianytko8.wixsite.com/alivia-nytko)       |PhD student |2024-present    |Co-advised with Joe Bailey |
|[Orlando Schwery](https://sites.google.com/site/orlandoschwery/home)      |PhD student |2014-2020    |                               |
|[Jenn Summers](https://cubalibrejenn.wixsite.com/jennsummers)             |PhD student |2020-2021 |                               |

## Undergrad students in my lab



|Name                                                                      |Stage     |Time in Lab |Note |
|:-------------------------------------------------------------------------|:---------|:-----------|:----|
|Tyler-Christian Daniels                                                   |Undergrad |2019-2020   |     |
|Stephen Ray                                                               |Undergrad |2019-2020   |     |
|Zach Tharpe                                                               |Undergrad |2019-2020   |     |
|[Christian Yarber](https://www.researchgate.net/profile/Christian_Yarber) |Undergrad |2015-2016   |     |
|[John Anderson](https://sites.google.com/view/john-anderson-research)	   |Undergrad |2021-present   |     |

## Faculty

Our department creates a mentoring committee for each new faculty member. I have been on the mentoring committee for the following faculty members:

|Name                                                          |Department |
|:-------------------------------------------------------------|:----------|
|[Liz Derryberry](https://derryberrylab.wordpress.com/people/) |EEB        |
|[Stephanie Kivlin](https://skivlin.wordpress.com/)            |EEB        |
|[Laura Russo](https://drlaurarusso.weebly.com)            |EEB        |
|[Kimberly Sheldon](https://www.biogeographyresearch.org/)      |EEB        |

# Service

Some of my service comes directly through publications (for example, DateLife allows people to easily get dated trees of life for teaching or research). Some comes through informal service (a colleague at a different institution reaching out for advice on a situation in their department, for example). Here I am listing more formal service.

* **Associate Head for EEB**: One of two associate heads under Head Susan Kalisz, my focus was on grad students and overall equity in the department. I also split interim head duties during a semester when the head was on sabbatical. My work included working with our grad admissions and grad affairs committees (including chairing the former one year while also serving as associate head), working on covid policy for the department (making sure everyone in a lab approved its safety protocols before it reopened, for example), and working with university lawyers on various issues (to be clear, none with me as a respondent or anything of the sort).
* **Associate Director for NIMBioS**: My focus was on postdocs, including vocational mentoring and building their social interactions.
* **President of the Society of Systematic Biologists**: This involves three years of commitment, as president-elect, president, and past president. During that time, I was involved in decisions on named awards, leadership and support of the society's two journals, a mentorship program, and more. I also served as de facto representative on the joint meeting committee for the Evolution meetings. I used my [presidential address](https://brianomeara.info/evol2023.html) to highlight four issues: how naming affects conservation; the need for and progress in handling misconduct in our field; issues with eponyms in our field; and our field's past and ongoing connection with eugenics (part of this included a new analysis of 122,096 papers and 141,284 unique authors).
* **Code of Conduct Committee for Evolution**: I have been on this committee since its inception. We built up support for an actionable code of conduct (I led an IRB-approved [study](https://figshare.com/articles/poster/Unwelcome_behaviors_at_the_Evolution_meetings_Survey_results/8311595?file=15572909) on the topic), developed a working code in association with various lawyers and other experts, interviewed and selected safety officers, and wrote up transparency reports based on incidents at the meeting.
* **Code of Ethics for the three societies**: The Society of Systematic Biologists, the Society for the Study of Evolution, and the American Society of Naturalists together worked on a shared code of ethics (to handle misconduct in general, not just at meetings). I gave feedback at various stages in the process and then helped more substantially in the final stages of getting it approved and selecting an investigatory agent for two of the societies (SSB and SSE).
* **Liason between joint meeting committee and the meeting location committee**: The Evolution meetings (annual meetings of 3-4 evolution societies with 1500-2000 attendees) have long been guided by California's travel ban policy in choosing locations (though with the time lead required between choosing a location and the meeting, many things can change). A group was tasked with creating recommendations for how to choose meetings in the future, including doing a survey; I act as a liason between that group and the joint meeting committee, and made recommendations on making their product more actionable.
* **Internal service**: Reviews of proposals and fellowships for our office of research and the grad school, workshops on applying for the NSF GRFP, review of another department, serving on an enhanced post-tenure review committee, dean's advisory committee, head search committee, college strategic planning committee for graduate education, and more.