---
author: brianomeara_ra6153
comments: false
date: 2015-07-18 00:46:01+00:00
layout: page
link: http://www.brianomeara.info/
slug: home
title: Home
wordpress_id: 397
---



<p><em>Most of this website is written for a specialist scientific audience; folks who aren’t scientists are very welcome to look around, but <a href="http://www.brianomeara.info/general">this page</a> might be a more accessible introduction. For a compressed version of information about me, please see my <a href="http://brianomeara.info/cv/">CV</a>.</em>
I am an associate professor at <a href="http://www.utk.edu/">U of Tennessee, Knoxville</a>, in the department of <a href="http://eeb.bio.utk.edu/">Ecology &amp; Evolutionary Biology</a>. In my lab, we develop and apply phylogenetic tools to address evolutionary questions.</p>
<p>[caption id=“attachment_960” align=“alignright” width=“361”][<img src="http://www.brianomeara.info/wp-content/uploads/2018/05/Screen-Shot-2018-05-23-at-8.51.11-AM.jpg" />](<a href="http://www.brianomeara.info/wp-content/uploads/2018/05/Screen-Shot-2018-05-23-at-8.51.11-AM.jpg" class="uri">http://www.brianomeara.info/wp-content/uploads/2018/05/Screen-Shot-2018-05-23-at-8.51.11-AM.jpg</a>) Diversification diagram using HiSSE (from Beaulieu &amp; O’Meara, 2016)[/caption]</p>
<p>They are usually generated by a direct research need: how can we tell whether this group is evolving at a different rate? How can we choose between phylogeographic models without limiting ourselves to a pre-selected small set? Is there hidden variation in states that lets some herbaceous plants retain the ability to make wood while others have lost this ability? By developing techniques to address these questions, we both solve the original question and enable other biologists to use these new techniques to answer more questions. Broadly, the areas covered include trait evolution, species delimitation, phylogeography, dating trees, and more work in progress.</p>
<div id="research" class="section level2">
<h2>Research</h2>
<p>I work on empirical systems in collaboration with others as well as other kinds of studies, but many people coming to this website are probably most interested in methods developed here. Some highlights:</p>
<ul>
<li><p><strong>Species delimitation</strong>: <a href="http://brianomeara.info/brownie">Brownie</a> (<a href="http://sysbio.oxfordjournals.org/content/early/2009/11/10/sysbio.syp077.short">O’Meara 2009</a>), which has nonparametric and parametric species delimitation: it returns a species tree and species assignments without first requiring assignment of individuals to populations, and <a href="http://www.phrapl.org">phrapl</a> (Jackson et al., 2017), which allows delimitation in the presence of gene flow.</p></li>
<li><p><strong>Continuous trait evolutionary rates</strong>: Different rates for Brownian motion were first in <a href="http://brianomeara.info/brownie">Brownie</a> (<a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fj.0014-3820.2006.tb01171.x">O’Meara, Ané, Sanderson, and Wainwright, 2006</a>), then RBrownie (no longer on CRAN), and are available in R in Liam Revell’s phytools. We later made flexible Ornstein-Uhlenbeck models that allow the mean, variance, and attraction parameters to change on the tree. This is in the R package <a href="http://cran.r-project.org/web/packages/OUwie/index.html">OUwie</a> (<a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2012.01619.x/abstract">Beaulieu, Jhwueng, Boettiger, and O’Meara, 2012</a>).</p></li>
<li><p><strong>Discrete trait models</strong>: Sometimes all you see is a binary state, but this hides a lot of complexity. For example, we can code plants as “woody” or “herbaceous”, but are all herbaceous plants the same? It’s possible that some still have the genes for producing wood and can readily re-evolve woodiness, but others cannot. Most methods treat these the same, but we allow hidden states: in this case, the hidden state might be “having genes for woodiness”. This is in the R package <a href="http://cran.r-project.org/web/packages/corHMM/index.html">corHMM</a> (<a href="http://sysbio.oxfordjournals.org/content/62/5/725.long">Beaulieu, O’Meara, and Donoghue 2013</a>).</p></li>
<li><p><strong>Discrete trait diversification models</strong>: BiSSE models are popular but can have issues with spurious correlations, as well as treating all taxa with a given observed trait as the same. We have what we think is a solution in <a href="http://cran.r-project.org/web/packages/hisse/index.html">HiSSE</a>.</p></li>
<li><p><strong>Phylogeography</strong>:Often in phylogeography we look at just a handful of models. In other cases, models must be restricted in some way, by, say, prohibiting migration. What if there were something like ModeTest but for phylogeography rather than nucleotide models? We have developed software, <a href="https://r-forge.r-project.org/projects/phrapl/">phrapl</a>, for this.</p></li>
</ul>
<p>[caption id=“attachment_1538” align=“alignright” width=“256”]<img src="http://www.brianomeara.info/wp-content/uploads/2018/10/movie645.gif" alt="Spinning diagram" /> PHRAPL model for three populations[/caption]</p>
<ul>
<li><p><strong>Dating</strong>: Stephen Smith and I worked on a reimplementation of Mike Sanderson’s penalized likelihood to scale to large trees: <a href="http://blackrim.org/programs/treepl/">treepl</a> (<a href="http://bioinformatics.oxfordjournals.org/content/28/20/2689.long">Smith and O’Meara 2012</a>). If you can get it installed, it works well. I am also working on <a href="http://www.datelife.org/">datelife.org</a>, a repository of chronograms inspired by TimeTree but with an eye for reusability.</p></li>
<li><p><strong>DNA/AA evolution</strong>: Read a recent phylogeny paper: they probably used GTR+G or a similar model, perhaps partitioned. We know far more about how DNA evolves than can be fit in a few parameter model, but a full, free codon model is generally too parameter-rich to fit, either. We have a method <a href="https://github.com/bomeara/selac">selac</a> that can combine models for mutation on nucleotides with selection on amino acids.</p></li>
<li><p><strong>In the pipeline</strong>: We have an improvement to the DEC model of Ree and Smith, approaches for doing comparative method on networks, rather than trees, ways to flexibly make complex models, and more, in prep. You can get a sense of this by looking at what <a href="http://www.brianomeara.info/people">people</a> in the lab are working on.</p></li>
</ul>
<p>See more info on research <a href="http://www.brianomeara.info/research">here</a>.</p>
</div>
<div id="teaching" class="section level2">
<h2>Teaching</h2>
<p>I’ve taught everything from 200-person introductory biology courses to small graduate reading groups. Every Fall I teach a macroevolution course for upper level undergraduate students and graduate students. I now teach (sponsored by an NSF CAREER grant) an <a href="http://phylometh.info">open graduate course in phylogenetic methods</a> every spring. I also organize and/or teach at a variety of workshops.
A key component of teaching is mentoring students and postdocs. I’ve mentored four graduate students and fifteen postdocs directly. See <a href="http://brianomeara.info/people/">here</a> for more information.</p>
<div id="service" class="section level3">
<h3>Service</h3>
<p>I have increasingly been involved in service. Part comes from organizing things like hackathons to draw in new participants and workshops to train biologists in new tools. I also co-organized the iEvoBio conference for three years and helped with other biology conferences (for example, scheduling all the lightning talks at a couple of Evolution meetings). I am also now an associate head for our EEB department with a focus on graduate students. I’ve also done workshops to help with students applying to grants, overseen grad admissions, etc. I have been heavily involved in the Society of Systematic Biologists, in the past serving as the communications director and a member of the council. I am also the associate director for postdoctoral activities at the National Institute for Biological and Mathematical Synthesis (NIMBioS), managing training and evaluation of postdocs. I am also the faculty advisor for Darwin Day Tennessee, one of the oldest Darwin Day events in the US (our activities now range from teacher workshops to bringing in speakers like Camile Parmesan and Neil Shubin to having giant puppets of Darwin and Wallace greet children at a birthday party for Darwin. I also try to make biology a better, more diverse place, whether it’s through helping sustain our department’s women in science group to talking about imposter syndrome with graduate students to finding ways to get a diverse set of applicants for our graduate program.</p>
</div>
</div>
<div id="funding" class="section level2">
<h2>Funding</h2>
<p>In the last eight years, work in the lab has been funded by five NSF grants to me as PI or Co-PI as well as awards from iPlant, Encyclopedia of Life, and Google Summer of Code to me or people doing work in the lab (see more info <a href="http://www.brianomeara.info/funding">here</a>). Grad students in the lab have been supported by teaching assistantships as well as a <a href="http://web.utk.edu/~peer/">PEER</a> fellowship. The UT Knoxville-based <a href="http://www.nimbios.org/">National Institute for Mathematical and Biological Synthesis (NIMBioS)</a> remains critically important for my work, whether by funding independent postdocs (I have mentored seven NIMBioS postdocs, in addition to three additional postdocs in my lab with other funds), sponsoring workshops, or organizing working groups.</p>
</div>
