<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>publications.utf8.md</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Brian O'Meara</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    CV, etc.
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="cv.html">CV</a>
    </li>
    <li>
      <a href="calendar.html">Calendar</a>
    </li>
    <li>
      <a href="feedback.html">Feedback</a>
    </li>
    <li>
      <a href="useful.html">Useful links</a>
    </li>
    <li>
      <a href="tenure.html">Materials for promotion to associate (and tenure)</a>
    </li>
    <li>
      <a href="promotion.html">Materials for promotion to full</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Lab
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="personnel.html">People</a>
    </li>
    <li>
      <a href="guidelines.html">Guidelines &amp; Recommendations</a>
    </li>
    <li>
      <a href="location.html">Location/Contact</a>
    </li>
    <li>
      <a href="opportunities.html">Opportunities</a>
    </li>
    <li>
      <a href="learningobjectives.html">Learning objectives</a>
    </li>
    <li>
      <a href="better-letter.html">BetterLetter</a>
    </li>
    <li>
      <a href="shouldyougotogradschool.html">Should you go to grad school to be a professor?</a>
    </li>
    <li>
      <a href="tenure.html">Materials for promotion to associate (and tenure)</a>
    </li>
    <li>
      <a href="promotion.html">Materials for promotion to full</a>
    </li>
    <li>
      <a href="general.html">Info for general public</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Teaching
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="teaching.html">Teaching overall</a>
    </li>
    <li>
      <a href="eeb464.html">Macroevolution (EEB464)</a>
    </li>
    <li>
      <a href="http://phylometh.info/">Phylogenetic Methods</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="aic.html">AIC</a>
    </li>
    <li>
      <a href="phylogenetics.html">Phylogenetics</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<div id="publications" class="section level2">
<h2>Publications</h2>
<p>I would recommend reading a few of my best papers to get a sense of my work; good examples are <a href="https://doi.org/10.1098/rspb.2015.2304">O’Meara, Smith, et al. (2016)</a> and <a href="https://doi.org/10.1093/sysbio/syw022">Beaulieu &amp; O’Meara (2016)</a>. Metrics are not as great (for example, my work with software might get cited than someone else’s work on a new discovery about evolution, but it is not clear that the former is ‘better’) , but I know some people find them useful, so: According to Google Scholar, my work has been cited 4234 times, and my h-index is 25. (Google Scholar tends to overestimate citations, however). If there were a journal of just my papers, its 2018 impact factor would be 11.909. Also note that I work under a very stringent criterion for when I get authorship – I have to actively make a significant contribution to the research and writing to merit authorship. For example, in 2015-6, three lab members had papers in <em>Science</em> (<a href="http://science.sciencemag.org/content/350/6264/1077.long">grad student Sam Borstein</a>, <a href="http://science.sciencemag.org/content/351/6268/28">postdoc Nick Matzke</a>, and <a href="http://science.sciencemag.org/content/353/6295/154.full">postdoc Sandy Kawano</a>) but I am, appropriately to my mind, not an author on any of these.</p>
<div id="papers" class="section level3">
<h3>Papers</h3>
<table>
<colgroup>
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th>Authors</th>
<th>Year</th>
<th>Title, Journal</th>
<th>DOI</th>
<th>Citations</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>JM Beaulieu, BC O’Meara</td>
<td>2019</td>
<td>“Diversity and skepticism are vital for comparative biology: a response to Donoghue and Edwards (2019)” NA 106 (5), 613-617</td>
<td><a href="https://doi.org/10.1002/ajb2.1278">10.1002/ajb2.1278</a></td>
<td>0</td>
</tr>
<tr class="even">
<td>JM Beaulieu, BC O’Meara, R Zaretzki, C Landerer, J Chai, MA Gilchrist</td>
<td>2019</td>
<td>“Population genetics based phylogenetics under stabilizing selection for an optimal amino acid sequence: A nested modeling approach” NA 36 (4), 834-851</td>
<td><a href="https://doi.org/10.1093/molbev/msy222">10.1093/molbev/msy222</a></td>
<td>2</td>
</tr>
<tr class="odd">
<td>SR Borstein, BC O’Meara</td>
<td>2018</td>
<td>“AnnotationBustR: An R package to extract subsequences from GenBank annotations” NA 6, e5179</td>
<td><a href="https://doi.org/10.7717/peerj.5179">10.7717/peerj.5179</a></td>
<td>1</td>
</tr>
<tr class="even">
<td>JM Beaulieu, BC O’Meara</td>
<td>2018</td>
<td>“Can we build it? Yes we can, but should we use it? Assessing the quality and value of a very large phylogeny of campanulid angiosperms” NA 105 (3), 417-432</td>
<td><a href="https://doi.org/10.1002/ajb2.1020">10.1002/ajb2.1020</a></td>
<td>12</td>
</tr>
<tr class="odd">
<td>JM Beaulieu, BC O’Meara</td>
<td>2018</td>
<td>“Can we build it? Yes we can, but should we use it? Assessing the quality and value of a very large phylogeny of campanulid angiosperms” NA 105 (3), 417-432</td>
<td><a href="https://doi.org/10.1002/ajb2.1020">10.1002/ajb2.1020</a></td>
<td>12</td>
</tr>
<tr class="even">
<td>DS Caetano, BC O’Meara, JM Beaulieu</td>
<td>2018</td>
<td>“Hidden state models improve state-dependent diversification approaches, including biogeographical models” NA 72 (11), 2308-2324</td>
<td><a href="https://doi.org/10.1111/evo.13602">10.1111/evo.13602</a></td>
<td>17</td>
</tr>
<tr class="odd">
<td>ND Jackson, AE Morales, BC Carstens, BC O’Meara</td>
<td>2017</td>
<td>“PHRAPL: Phylogeographic Inference Using Approximate Likelihoods” NA 66 (6), 1045-1053</td>
<td><a href="https://doi.org/10.1093/sysbio/syx001">10.1093/sysbio/syx001</a></td>
<td>22</td>
</tr>
<tr class="even">
<td>BC Carstens, AE Morales, ND Jackson, BC O’Meara</td>
<td>2017</td>
<td>“Objective choice of phylogeographic models” NA 116, 136-140</td>
<td><a href="https://doi.org/10.1016/j.ympev.2017.08.018">10.1016/j.ympev.2017.08.018</a></td>
<td>3</td>
</tr>
<tr class="odd">
<td>AE Morales, ND Jackson, TA Dewey, BC O’Meara, BC Carstens</td>
<td>2017</td>
<td>“Speciation with gene flow in North American Myotis bats” NA 66 (3), 440-452</td>
<td><a href="https://doi.org/10.1093/sysbio/syw100">10.1093/sysbio/syw100</a></td>
<td>30</td>
</tr>
<tr class="even">
<td>ND Jackson, BC Carstens, AE Morales, BC O’Meara</td>
<td>2017</td>
<td>“Species delimitation with gene flow” NA 66 (5), 799-812</td>
<td><a href="https://doi.org/10.1093/sysbio/syw117">10.1093/sysbio/syw117</a></td>
<td>49</td>
</tr>
<tr class="odd">
<td>JM Bosco, SE Riechert, BC O’Meara</td>
<td>2017</td>
<td>“The ontogeny of personality traits in the desert funnel-web spider, Agelenopsis lisa (Araneae: Agelenidae)” NA 123 (9), 648-658</td>
<td><a href="https://doi.org/10.1111/eth.12639">10.1111/eth.12639</a></td>
<td>1</td>
</tr>
<tr class="even">
<td>O Schwery, BC O’Meara</td>
<td>2016</td>
<td>“MonoPhy : a simple R package to find and visualize monophyly issues” NA 2, e56</td>
<td><a href="https://doi.org/10.7717/peerj-cs.56">10.7717/peerj-cs.56</a></td>
<td>8</td>
</tr>
<tr class="odd">
<td>O Schwery, BC O’Meara</td>
<td>2016</td>
<td>“MonoPhy: A simple R package to find and visualize monophyly issues” NA 2, e56</td>
<td><a href="https://doi.org/10.7717/peerj-cs.56">10.7717/peerj-cs.56</a></td>
<td>8</td>
</tr>
<tr class="even">
<td>JM Beaulieu, BC O’Meara</td>
<td>2016</td>
<td>“Detecting hidden diversification shifts in models of trait-dependent speciation and extinction” NA 65 (4), 583-601</td>
<td><a href="https://doi.org/10.1093/sysbio/syw022">10.1093/sysbio/syw022</a></td>
<td>152</td>
</tr>
<tr class="odd">
<td>BC O’Meara, SD Smith, WS Armbruster, LD Harder, CR Hardy, …</td>
<td>2016</td>
<td>“Non-equilibrium dynamics and floral trait interactions shape extant angiosperm diversity” NA 283 (1830), 20152304</td>
<td><a href="https://doi.org/10.1098/rspb.2015.2304">10.1098/rspb.2015.2304</a></td>
<td>31</td>
</tr>
<tr class="even">
<td>BC O’Meara, JM Beaulieu</td>
<td>2016</td>
<td>“Past, future, and present of state-dependent models of diversification” NA 103 (5), 792-795</td>
<td><a href="https://doi.org/10.3732/ajb.1600012">10.3732/ajb.1600012</a></td>
<td>18</td>
</tr>
<tr class="odd">
<td></td>
<td>2015</td>
<td>“Erratum: Three keys to the radiation of angiosperms into freezing environments (Nature (2014) 506 (89-92) (<a href="Doi:10.1038/nature12872" class="uri">Doi:10.1038/nature12872</a>))” NA</td>
<td><a href="https://doi.org/10.1038/nature14371">10.1038/nature14371</a></td>
<td>NA</td>
</tr>
<tr class="even">
<td>BC O’Meara, KL Graham, SM Pellis, GM Burghardt</td>
<td>2015</td>
<td>“Evolutionary models for the retention of adult–adult social play in primates: The roles of diet and other factors associated with resource acquisition” NA 23 (6), 381-391</td>
<td><a href="https://doi.org/10.1177/1059712315611733">10.1177/1059712315611733</a></td>
<td>8</td>
</tr>
<tr class="odd">
<td>JM Beaulieu, BC O’Meara</td>
<td>2015</td>
<td>“Extinction can be estimated from moderately sized molecular phylogenies” NA 69 (4), 1036-1043</td>
<td><a href="https://doi.org/10.1111/evo.12614">10.1111/evo.12614</a></td>
<td>69</td>
</tr>
<tr class="even">
<td>JM Beaulieu, BC O’Meara, P Crane, MJ Donoghue</td>
<td>2015</td>
<td>“Heterogeneous rates of molecular evolution and diversification could explain the triassic age estimate for angiosperms” NA 64 (5), 869-878</td>
<td><a href="https://doi.org/10.1093/sysbio/syv027">10.1093/sysbio/syv027</a></td>
<td>59</td>
</tr>
<tr class="odd">
<td>MSP Aldrovandi, JE Johnson, B O’Meara, RH Petersen, KW Hughes</td>
<td>2015</td>
<td>“The Xeromphalina campanella/kauffmanii complex: Species delineation and biogeographical patterns of speciation” NA 107 (6), 1270-1284</td>
<td><a href="https://doi.org/10.3852/15-087">10.3852/15-087</a></td>
<td>5</td>
</tr>
<tr class="even">
<td>AE Zanne, DC Tank, WK Cornwell, JM Eastman, SA Smith, RG FitzJohn, …</td>
<td>2015</td>
<td>“Zanne et al. reply” NA 521 (7552), E6</td>
<td><a href="https://doi.org/10.1038/nature14394">10.1038/nature14394</a></td>
<td>1</td>
</tr>
<tr class="odd">
<td></td>
<td>2014</td>
<td>“Erratum: Three keys to the radiation of angiosperms into freezing environments (Nature (2014) 506 (89-92) <a href="DOI:10.1038/nature12872" class="uri">DOI:10.1038/nature12872</a>)” NA</td>
<td><a href="https://doi.org/10.1038/nature13842">10.1038/nature13842</a></td>
<td>NA</td>
</tr>
<tr class="even">
<td>WK Cornwell, M Westoby, DS Falster, RG FitzJohn, BC O’Meara, …</td>
<td>2014</td>
<td>“Functional distinctiveness of major plant lineages” NA 102 (2), 345-356</td>
<td><a href="https://doi.org/10.1111/1365-2745.12208">10.1111/1365-2745.12208</a></td>
<td>81</td>
</tr>
<tr class="odd">
<td>DC Jhwueng, S Huzurbazar, BC O’Meara, L Liu</td>
<td>2014</td>
<td>“Investigating the performance of AIC in selecting phylogenetic models” NA 13 (4), 459-475</td>
<td><a href="https://doi.org/10.1515/sagmb-2013-0048">10.1515/sagmb-2013-0048</a></td>
<td>5</td>
</tr>
<tr class="even">
<td>BL Banbury, BC O’Meara</td>
<td>2014</td>
<td>“Reol: R interface to the encyclopedia of life” NA 4 (12), 2577-2583</td>
<td><a href="https://doi.org/10.1002/ece3.1109">10.1002/ece3.1109</a></td>
<td>3</td>
</tr>
<tr class="odd">
<td>JH Williams, ML Taylor, BC O’Meara</td>
<td>2014</td>
<td>“Repeated evolution of tricellular (and bicellular) pollen” NA 101 (4), 559-571</td>
<td><a href="https://doi.org/10.3732/ajb.1300423">10.3732/ajb.1300423</a></td>
<td>27</td>
</tr>
<tr class="even">
<td>AE Zanne, DC Tank, WK Cornwell, JM Eastman, SA Smith, RG FitzJohn, …</td>
<td>2014</td>
<td>“Three keys to the radiation of angiosperms into freezing environments” NA 506 (7486), 89</td>
<td><a href="https://doi.org/10.1038/nature12872">10.1038/nature12872</a></td>
<td>575</td>
</tr>
<tr class="odd">
<td>JM Beaulieu, BC O’Meara, MJ Donoghue</td>
<td>2013</td>
<td>“Identifying hidden rate changes in the evolution of a binary morphological character: The evolution of plant habit in campanulid angiosperms” NA 62 (5), 725-737</td>
<td><a href="https://doi.org/10.1093/sysbio/syt034">10.1093/sysbio/syt034</a></td>
<td>144</td>
</tr>
<tr class="even">
<td>C Darrin Hulsey, BP Keck, H Alamillo, BC O’Meara</td>
<td>2013</td>
<td>“Mitochondrial genome primers for Lake Malawi cichlids” NA 13 (3), 347-353</td>
<td><a href="https://doi.org/10.1111/1755-0998.12066">10.1111/1755-0998.12066</a></td>
<td>12</td>
</tr>
<tr class="odd">
<td>DE Soltis, ME Mort, M Latvis, EV Mavrodiev, BC O’Meara, PS Soltis, …</td>
<td>2013</td>
<td>“Phylogenetic relationships and character evolution analysis of Saxifragales using a supermatrix approach” NA 100 (5), 916-929</td>
<td><a href="https://doi.org/10.3732/ajb.1300044">10.3732/ajb.1300044</a></td>
<td>51</td>
</tr>
<tr class="even">
<td>A Stoltzfus, H Lapp, N Matasci, H Deus, B Sidlauskas, CM Zmasek, …</td>
<td>2013</td>
<td>“Phylotastic! Making tree-of-life knowledge accessible, reusable and convenient” NA 14 (1), 158</td>
<td><a href="https://doi.org/10.1186/1471-2105-14-158">10.1186/1471-2105-14-158</a></td>
<td>26</td>
</tr>
<tr class="odd">
<td>BC O’Meara</td>
<td>2012</td>
<td>“Evolutionary inferences from phylogenies: a review of methods” NA 43, 267-285</td>
<td><a href="https://doi.org/10.1146/annurev-ecolsys-110411-160331">10.1146/annurev-ecolsys-110411-160331</a></td>
<td>152</td>
</tr>
<tr class="even">
<td>JM Beaulieu, DC Jhwueng, C Boettiger, BC O’Meara</td>
<td>2012</td>
<td>“Modeling stabilizing selection: Expanding the Ornstein-Uhlenbeck model of adaptive evolution” NA 66 (8), 2369-2383</td>
<td><a href="https://doi.org/10.1111/j.1558-5646.2012.01619.x">10.1111/j.1558-5646.2012.01619.x</a></td>
<td>328</td>
</tr>
<tr class="odd">
<td>A Stoltzfus, B O’meara, J Whitacre, R Mounce, EL Gillespie, S Kumar, …</td>
<td>2012</td>
<td>“Sharing and re-use of phylogenetic trees (and associated data) to facilitate synthesis” NA 5 (1), 574</td>
<td><a href="https://doi.org/10.1186/1756-0500-5-574">10.1186/1756-0500-5-574</a></td>
<td>40</td>
</tr>
<tr class="even">
<td>SA Smith, BC O’Meara</td>
<td>2012</td>
<td>“TreePL: Divergence time estimation using penalized likelihood for large phylogenies” NA 28 (20), 2689-2690</td>
<td><a href="https://doi.org/10.1093/bioinformatics/bts492">10.1093/bioinformatics/bts492</a></td>
<td>158</td>
</tr>
<tr class="odd">
<td>JC Stack, LJ Harmon, B O’Meara</td>
<td>2011</td>
<td>“RBrownie: an R package for testing hypotheses about rates of evolutionary change” NA 2 (6), 660-662</td>
<td><a href="https://doi.org/10.1111/j.2041-210x.2011.00112.x">10.1111/j.2041-210x.2011.00112.x</a></td>
<td>11</td>
</tr>
<tr class="even">
<td>JM Abercrombie, BC O’Meara, AR Moffatt, JH Williams</td>
<td>2011</td>
<td>“Developmental evolution of flowering plant pollen tube cell walls: Callose synthase (CalS) gene expression patterns” NA 2 (1), 14</td>
<td><a href="https://doi.org/10.1186/2041-9139-2-14">10.1186/2041-9139-2-14</a></td>
<td>30</td>
</tr>
<tr class="odd">
<td>SA Goff, M Vaughn, S McKay, E Lyons, AE Stapleton, D Gessler, …</td>
<td>2011</td>
<td>“The iPlant collaborative: Cyberinfrastructure for plant biology” NA 2, 34</td>
<td><a href="https://doi.org/10.3389/fpls.2011.00034">10.3389/fpls.2011.00034</a></td>
<td>370</td>
</tr>
<tr class="even">
<td>DC Collar, JA Schulte, BC O’meara, JB Losos</td>
<td>2010</td>
<td>“Habitat use affects morphological diversification in dragon lizards” NA 23 (5), 1033-1049</td>
<td><a href="https://doi.org/10.1111/j.1420-9101.2010.01971.x">10.1111/j.1420-9101.2010.01971.x</a></td>
<td>60</td>
</tr>
<tr class="odd">
<td>BC O’Meara</td>
<td>2010</td>
<td>“New heuristic methods for joint species delimitation and species tree inference” NA 59 (1), 59-73</td>
<td><a href="https://doi.org/10.1093/sysbio/syp077">10.1093/sysbio/syp077</a></td>
<td>247</td>
</tr>
<tr class="even">
<td>SA Smith, BC O’Meara</td>
<td>2009</td>
<td>“Morphogenera, monophyly, and macroevolution” NA 106 (36), E97-E98</td>
<td><a href="https://doi.org/10.1073/pnas.0906918106">10.1073/pnas.0906918106</a></td>
<td>4</td>
</tr>
<tr class="odd">
<td>DC Collar, BC O’Meara, PC Wainwright, TJ Near</td>
<td>2009</td>
<td>“Piscivory limits diversification of feeding morphology in centrarchid fishes” NA 63 (6), 1557-1573</td>
<td><a href="https://doi.org/10.1111/j.1558-5646.2009.00626.x">10.1111/j.1558-5646.2009.00626.x</a></td>
<td>122</td>
</tr>
<tr class="even">
<td>BC O’Meara, C Ané, MJ Sanderson, PC Wainwright</td>
<td>2006</td>
<td>“Testing for different rates of continuous trait evolution using likelihood” NA 60 (5), 922-933</td>
<td><a href="https://doi.org/10.1554/05-130.1">10.1554/05-130.1</a></td>
<td>525</td>
</tr>
<tr class="odd">
<td>AC Driskell, C Ané, JG Burleigh, MM McMahon, BC O’Meara, …</td>
<td>2004</td>
<td>“Prospects for building the tree of life from large sequence databases” NA 306 (5699), 1172-1174</td>
<td><a href="https://doi.org/10.1126/science.1102036">10.1126/science.1102036</a></td>
<td>249</td>
</tr>
<tr class="even">
<td>BD Farrell, AS Sequeira, BC O’Meara, BB Normark, JH Chung, BH Jordal</td>
<td>2001</td>
<td>“The evolution of agriculture in beetles (Curculionidae: Scolytinae and Platypodinae)” NA 55 (10), 2011-2027</td>
<td><a href="https://doi.org/"></a></td>
<td>357</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="publications-books-or-book-chapters" class="section level2">
<h2>Publications: Books or Book Chapters</h2>
<table>
<colgroup>
<col width="25%" />
<col width="25%" />
<col width="25%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Authors</th>
<th>Year</th>
<th>Title</th>
<th>Book</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>B O’Meara</td>
<td>2016</td>
<td>“Phylogenetic Comparative Method”</td>
<td>Encyclopedia of Evolutionary Biology</td>
</tr>
<tr class="even">
<td>JM Beaulieu, BC O’Meara</td>
<td>2014</td>
<td>“Hidden Markov Models for Studying the Evolution of Binary Morphological Characters”</td>
<td>Modern Phylogenetic Comparative Methods and Their Application in Evolutionary Biology: Concepts and Practice</td>
</tr>
<tr class="odd">
<td>BC O’Meara, JM Beaulieu</td>
<td>2014</td>
<td>“Modelling Stabilizing Selection: The Attraction of Ornstein–Uhlenbeck Models”</td>
<td>Modern Phylogenetic Comparative Methods and Their Application in Evolutionary Biology: Concepts and Practice</td>
</tr>
</tbody>
</table>
<div id="packages" class="section level3">
<h3>Packages</h3>
<p>Software developed in R (often associated with a publication, but not always)</p>
<table>
<colgroup>
<col width="25%" />
<col width="25%" />
<col width="25%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Package</th>
<th>Authors</th>
<th>Description</th>
<th>Downloads</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>AnnotationBustR</td>
<td>Samuel R. Borstein <a href="mailto:sam@borstein.com" class="email">sam@borstein.com</a>, Brian O’Meara <a href="mailto:bomeara@utk.edu" class="email">bomeara@utk.edu</a></td>
<td>Extract Subsequences from GenBank Annotations</td>
<td>9684</td>
</tr>
<tr class="even">
<td>BMhyb</td>
<td>Dwueng-Chwuan Jhwueng, Brian C. O’Meara</td>
<td>Comparative Methods for Phylogenetic Networks</td>
<td>7579</td>
</tr>
<tr class="odd">
<td>BMhyd</td>
<td>Dwueng-Chwuan Jhwueng <a href="mailto:djhwueng@umail.iu.edu" class="email">djhwueng@umail.iu.edu</a> and Brian O’Meara <a href="mailto:omeara.brian@gmail.com" class="email">omeara.brian@gmail.com</a></td>
<td>PCM for Hybridization</td>
<td>17822</td>
</tr>
<tr class="even">
<td>corHMM</td>
<td>Jeremy M. Beaulieu <a href="mailto:jbeaulieu@nimbios.org" class="email">jbeaulieu@nimbios.org</a>, Jeffrey C. Oliver, <a href="mailto:jeffreycoliver@gmail.com" class="email">jeffreycoliver@gmail.com</a>, Brian O’Meara <a href="mailto:bomeara@utk.edu" class="email">bomeara@utk.edu</a></td>
<td>Analysis of Binary Character Evolution</td>
<td>37223</td>
</tr>
<tr class="odd">
<td>hisse</td>
<td>Jeremy Beaulieu, Brian O’Meara, Daniel Caetano</td>
<td>Hidden State Speciation and Extinction</td>
<td>30814</td>
</tr>
<tr class="even">
<td>OUwie</td>
<td>Jeremy M. Beaulieu <a href="mailto:jbeaulieu@nimbios.org" class="email">jbeaulieu@nimbios.org</a>, Brian O’Meara <a href="mailto:bomeara@utk.edu" class="email">bomeara@utk.edu</a></td>
<td>Analysis of Evolutionary Rates in an OU Framework</td>
<td>38672</td>
</tr>
<tr class="odd">
<td>phylobase</td>
<td>R Hackathon et al. (alphabetically: Ben Bolker, Marguerite Butler, Peter Cowan, Damien de Vienne, Dirk Eddelbuettel, Mark Holder, Thibaut, Jombart, Steve Kembel, Francois Michonneau, David Orme, Brian O’Meara, Emmanuel Paradis, Jim Regetz, Derrick Zwickl)</td>
<td>Base Package for Phylogenetic Structures and Comparative Data</td>
<td>114299</td>
</tr>
<tr class="even">
<td>rncl</td>
<td>Francois Michonneau, (<a href="https://orcid.org/0000-0002-9092-966X" class="uri">https://orcid.org/0000-0002-9092-966X</a>), Ben Bolker (<a href="https://orcid.org/0000-0002-2127-0443" class="uri">https://orcid.org/0000-0002-2127-0443</a>), Mark Holder (<a href="https://orcid.org/0000-0001-5575-0536" class="uri">https://orcid.org/0000-0001-5575-0536</a>), Paul Lewis (<a href="https://orcid.org/0000-0001-9852-8759" class="uri">https://orcid.org/0000-0001-9852-8759</a>), Brian O’Meara (<a href="https://orcid.org/0000-0002-0337-5997" class="uri">https://orcid.org/0000-0002-0337-5997</a>)</td>
<td>An Interface to the Nexus Class Library</td>
<td>115812</td>
</tr>
<tr class="odd">
<td>selac</td>
<td>Jeremy Beaulieu, JJ Chai, Mike Gilchrist, Cedric Landerer, Graham Derryberry, Brian O’Meara</td>
<td>Selection Models for Amino Acid and/or Codon Evolution</td>
<td>1507</td>
</tr>
</tbody>
</table>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
