<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/header-attrs-2.9/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Brian O'Meara</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    CV, etc.
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="cv.html">CV</a>
    </li>
    <li>
      <a href="calendar.html">Calendar</a>
    </li>
    <li>
      <a href="feedback.html">Feedback</a>
    </li>
    <li>
      <a href="useful.html">Useful links</a>
    </li>
    <li>
      <a href="tenure.html">Materials for promotion to associate (and tenure)</a>
    </li>
    <li>
      <a href="promotion.html">Materials for promotion to full</a>
    </li>
    <li>
      <a href="SSB.html">Candidacy for SSB president</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Lab
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="personnel.html">People</a>
    </li>
    <li>
      <a href="guidelines.html">Guidelines &amp; Recommendations</a>
    </li>
    <li>
      <a href="location.html">Location/Contact</a>
    </li>
    <li>
      <a href="opportunities.html">Opportunities</a>
    </li>
    <li>
      <a href="learningobjectives.html">Learning objectives</a>
    </li>
    <li>
      <a href="better-letter.html">BetterLetter</a>
    </li>
    <li>
      <a href="shouldyougotogradschool.html">Should you go to grad school to be a professor?</a>
    </li>
    <li>
      <a href="tenure.html">Materials for promotion to associate (and tenure)</a>
    </li>
    <li>
      <a href="promotion.html">Materials for promotion to full</a>
    </li>
    <li>
      <a href="general.html">Info for general public</a>
    </li>
    <li>
      <a href="covid.html">COVID</a>
    </li>
    <li>
      <a href="schedule.html">EEB scheduling</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Teaching
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="teaching.html">Teaching overall</a>
    </li>
    <li>
      <a href="eeb464.html">Macroevolution (EEB464)</a>
    </li>
    <li>
      <a href="http://phylometh.info/">Phylogenetic Methods</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="aic.html">AIC</a>
    </li>
    <li>
      <a href="phylogenetics.html">Phylogenetics</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<p>Note you can see a feed of recent articles at <a href="http://scholar.google.com/citations?sortby=pubdate&amp;hl=en&amp;user=vpjEkQwAAAAJ&amp;view_op=list_works">Google Scholar</a>; this updates more quickly than this page does. In the lag time between submission and publication, you can see my manuscripts on <a href="http://biorxiv.org/search/author1%3ABrian%2BO%2527Meara%2B">bioRxiv</a>.</p>
<div id="trait-evolution" class="section level3">
<h3>Trait evolution</h3>
<p>Work in our lab most usually focuses on trait evolution, with most emphasis on methods that allow heterogeneity over the tree. For example, O’Meara et al. (2006) developed a method that allows the rate of Brownian motion to vary over a tree. This can be used to test ideas about rate evolution (Collar et al. 2009; Collar et al. 2010) and has been cited 150 times (according to Google Scholar in Aug. 2012). This work has been extended to deal with shifting rates, strengths of attraction, and means under an Ornstein-Uhlenbeck process (Beaulieu et al. 2012). Similarly, we have developed a model for different rates of gene loss on different parts of a tree (O’Meara 2007) and methods for looking at hidden rates for discrete characters (Beaulieu et al. 2014). A review of methods in general has been published (O’Meara 2012b) where I tried to show connections across methods. We have also looked at empirical datasets, often plants, to understand evolutionary processes (eg., Zanne et al. 2013, Williams et al. 2014, Cornwell et al. 2014).</p>
</div>
<div id="species-delimitation-gene-tree-species-tree-messiness" class="section level3">
<h3>Species delimitation / gene tree-species tree messiness</h3>
<p>The fact that gene trees and containing species trees has become increasingly apparent through work by many researchers. While this is often seen as a problem, our lab used information from gene tree-species tree conflict to estimate species boundaries and a species tree simultaneously, without requiring a priori specification of potential groups (O’Meara 2010). Ongoing work in collaboration with Bryan Carstens has resulted in a draft program to broaden this approach to phylogeography.</p>
</div>
<div id="phylogenetic-inference" class="section level3">
<h3>Phylogenetic inference</h3>
<p>To do something useful with a phylogeny first requires getting the phylogeny, perhaps with dates. Various publications have come from inferring trees (Driskell et al. 2004; Abercrombie et al. 2011), developing new implementations for dating trees (Smith and O’Meara 2012), and defending the need for trees (Smith and O’Meara 2009).</p>
</div>
<div id="software" class="section level3">
<h3>Software</h3>
<p>The point of what we do is to foster new discoveries about biology, especially about macroevolutionary processes. Availability of methods seems to be rate-limiting in many cases: once a new method appears, a flurry of empirical activity uses it to test long-standing hypotheses or infer new patterns. For this to happen, methods have to be available in software. Our work with iPlant (Goff et al. 2011) has aimed to make comparative methods more scalable and accessible, through the iPlant discovery environment and through our recent work on the rPlant interface to iPlant tools. Work in R in general has become a recent trend, ranging from porting existing code to R (Stack et al. 2011) to maintaining a task view of all R packages that deal with phylogenetics (O’Meara 2012a). See the <a href="http://www.brianomeara.info/software">software page</a> for more.</p>
</div>
<div id="selected-publications" class="section level3">
<h3>Selected publications</h3>
<p>Abercrombie, J. M., B. C. O’Meara, A. R. Moffatt, and J. H. Williams. 2011. Developmental evolution of flowering plant pollen tube cell walls: callose synthase (CalS) gene expression patterns. EvoDevo 2:14. Beaulieu, J. M., D.-C. Jhwueng, C. Boettiger, and B. C. O’Meara. 2012. Modeling stabilizing selection: Expanding the Ornstein- Uhlenbeck model of adaptive evolution. Evolution 66 (8), 2369-2383 Beaulieu, J.M., B.C. O’Meara, and M.J. Donoghue. 2014. Identifying hidden rate changes in the evolution of a binary morphological character: the evolution of plant habit in campanulid angiosperms. Systematic biology 62 (5), 725-737 Collar, D. C., B. C. O’Meara, P. C. Wainwright, and T. J. Near. 2009. Piscivory Limits Diversification of Feeding Morphology in Centrarchid Fishes. Evolution 63:1557-1573. Collar, D. C., J. A. Schulte, B. C. O’Meara, and J. B. Losos. 2010. Habitat use affects morphological diversification in dragon lizards. Journal of Evolutionary Biology 23:1033-1049. WK Cornwell, M Westoby, DS Falster, RG FitzJohn, BC O’Meara, M.W. Pennell, D.J. McGlinn, J.M. Eastman, A.T. Moles, P.B. Reich, D.C. Tank, I.J. Wright, L. Aarssen, J.M. Beaulieu, R.M. Kooyman, M.R. Leishman, E.T. Miller, Ü. Niinemets, J. Oleksyn, A. Ordonez, D.L. Royer, S.A. Smith, P.F. Stevens, L. Warman, P. Wilf, and A.E. Zanne 2014. Functional distinctiveness of major plant lineages. Journal of Ecology 102 (2), 345-356 Driskell, A. C., C. Ane, J. G. Burleigh, M. M. McMahon, B. C. O’Meara, and M. J. Sanderson. 2004. Prospects for building the tree of life from large sequence databases. Science 306:1172-1174. Goff, S., A., M. Vaughn, S. McKay, E. Lyons, E. Stapleton Ann, D. Gessler, N. Matasci, L. Wang, M. Hanlon, A. Lenards, A. Muir, N. Merchant, S. Lowry, S. Mock, M. Helmke, A. Kubach, M. Narro, N. Hopkins, D. Micklos, U. Hilgert, M. Gonzales, C. Jordan, E. Skidmore, R. Dooley, J. Cazes, R. McLay, Z. Lu, S. Pasternak, L. Koesterke, H. Piel William, R. Grene, C. Noutsos, K. Gendler, X. Feng, C. Tang, M. Lent, S.-j. Kim, K. Kvilekval, B. S. Manjunath, V. Tannen, A. Stamatakis, M. Sanderson, M. Welch Stephen, K. Cranston, P. Soltis, D. Soltis, B. C. O’Meara, C. Ane, T. Brutnell, J. Kleibenstein Daniel, W. White Jeffrey, J. Leebens-Mack, J. Donoghue Michael, P. Spalding Edgar, J. Vision Todd, R. Myers Christopher, D. Lowenthal, J. Enquist Brian, B. Boyle, A. Akoglu, G. Andrews, S. Ram, D. Ware, L. Stein, and D. Stanzione. 2011. The iPlant Collaborative: Cyberinfrastructure for Plant Biology. Frontiers in Plant Science 2. O’Meara, B. C. 2007. Estimating Different Rates of Gene Loss on a Tree. Genetics 177:1415-1416. O’Meara, B. C. 2010. New Heuristic Methods for Joint Species Delimitation and Species Tree Inference. Systematic Biology 59:59-73. O’Meara, B. C. 2012a. CRAN Task View: Phylogenetics. Version 2012-02-02, URL <a href="http://cran.r-project.org/web/views/Phylogenetics.html" class="uri">http://cran.r-project.org/web/views/Phylogenetics.html</a>. O’Meara, B. C. 2012b. Evolutionary Inferences from Phylogenies: A Review of Methods. Annual Review of Ecology, Evolution, and Systematics 43. O’Meara, B. C., C. Ane, M. J. Sanderson, and P. C. Wainwright. 2006. Testing for different rates of continuous trait evolution using likelihood. Evolution 60:922-933. Smith, S. A., and B. C. O’Meara. 2009. Morphogenera, monophyly, and macroevolution. Proceedings of the National Academy of Sciences of the United States of America 106:E97-E98. Smith, S. A., and B. C. O’Meara. 2012. treePL: Divergence time estimation using penalized likelihood for large phylogenies. Bioinformatics. Stack, J. C., L. J. Harmon, and B. O’Meara. 2011. RBrownie: an R package for testing hypotheses about rates of evolutionary change. Methods in Ecology and Evolution 2:660-662. Williams, J.H., M.L. Taylor, and B.C. O’Meara. 2014. Repeated evolution of tricellular (and bicellular) pollen. American journal of botany 101 (4), 559-571 Zanne, A.E., D.C. Tank, W.K. Cornwell, J.M. Eastman, S.A. Smith, R.G. FitzJohn, D.J. McGlinn, B.C. O’Meara, A.T. Moles, P.B. Reich, D.L. Royer, D.E. Soltis, P.F. Stevens, M. Westoby, I.J. Wright, L. Aarssen, R.I. Bertin, A. Calaminus, R. Govaerts, F. Hemmings, M.R. Leishman, J. Oleksyn, P.S. Soltis, N.G. Swenson, L. Warman, J.M. Beaulieu. 2013. Three keys to the radiation of angiosperms into freezing environments. Nature.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
